src/Database.ts

```diff
@@ -10,18 +10,17 @@ export class Database extends (BaseDatabase as any)
         super(connection);
     }
 
-    public resources(): any
+    public resources(): Array<Resource>
     {
         const resources: Array<Resource> = [];
-
         for (const entityMetadata of this.connection.entityMetadatas)
-            resources.push(new Resource(entityMetadata.target as any) as any);
+            resources.push(new Resource(entityMetadata.target as typeof BaseEntity));
 
         return resources;
     }
 
-    public static isAdapterFor(database: any)
-    {
-        return database instanceof Connection;
```

Comment:
I had some issues with `instanceonf` with other adapters. Namely, when you are running app in the dev environment all `devDependencies` are installed, and an entire application has different versions of typeorm package and instanceof doesn't work. On production typeorm is installed as a peerDependency (only one version exists) and then it works.

That is why I changed that to checking if passed object has `entityMetadatas` property